website/components/remote-plugin-docs/server.js

```diff
@@ -0,0 +1,75 @@
+import fs from 'fs'
+import path from 'path'
+import {
+  getNodeFromPath,
+  getPathsFromNavData,
+  validateNavData,
+} from '@hashicorp/react-docs-page/server'
+import renderPageMdx from '@hashicorp/react-docs-page/render-page-mdx'
```

Comment:
Currently, Packer remote plugin docs are being treated as an individual use case.

To meet this use case, we've built on the [revised DocsPage and DocsSidenav work we've done in implementing MKTG-032](https://github.com/hashicorp/react-components/pull/154).

This `server.js` file composes functions used in the revised DocsPage `server.js` utility, and does additional work, namely fetching and merging the remote plugin docs NavNodes.